lib/js_defer.js

```diff
@@ -1456,6 +1462,29 @@ deferJsNs.DeferJs.prototype.noDeferCreateElementOverride = function() {
   };
 };
 
+deferJsNs.DeferJs.prototype.deferInsertScriptTag = function() {
+  var me = this;
+  Element.prototype.insertBefore = function (newElement, refElement) {
+    if (me.overrideDefaultImplementation_ &&
+      newElement.nodeName === 'SCRIPT' &&
+      newElement.getAttribute('src')) {
+        me.log('Defer insertion of ' + newElement.getAttribute('src'));
+        me.originInsertBeforeQueue_.push([this, newElement, refElement]);
+    } else {
+      me.originInsertBefore_.apply(this, arguments);
+    }
+  };
+}
+
+deferJsNs.DeferJs.prototype.insertScriptInQueue = function() {
+  var me = this;
+
+  this.originInsertBeforeQueue_.forEach(function(call) {
+    me.log('Dynamically insert: ' + call[1].getAttribute('src'));
+    me.originInsertBefore_.call(call[0], call[1], call[2]);
```

Comment:
Is `insertBefore` really needed here? The insertion position in the DOM lost all meaning if the insertion is deferred.